### PR TITLE
Clarify Next steps

### PR DESCRIPTION
**Pull request by mjahoda**
https://github.com/redhat-documentation/modular-docs/issues/200

### CODE REVIEW

File: modular-docs-manual/content/topics/creating_procedure_modules.adoc [hunk 1]
```diff
@@ -41,6 +41,9 @@ This section is optional. The additional resources list links to other material
 
 Focus on relevant resources that might interest the user. Do not list resources for completeness. If a resource applies to all of the modules in a user story, consider listing the resource in the Additional resources section of the assembly file. If you do this, consider including the resource in the procedure module as a comment.
 
+.Procedure next steps
+This section is optional. You can use it for pointing the reader to instructions covering tasks that might follow after the current procedure. Place this section after the Additional resources section with the *Next steps* heading. Alternatively, if the user should perform the next steps before the verification steps, place the section in between the Procedure section and the Verification steps section.
```
Comment (thatdocslady) [inline modular-docs-manual/content/topics/creating_procedure_modules.adoc:45]:
```suggestion
This section is optional. You can use it for pointing the reader to tasks that might follow after the current procedure. Place this section after the *Additional resources* section and use the *Next steps* heading. Alternatively, if the user needs to perform the next steps before the verification steps, place the section between the *Procedure* section and the *Verification* section.
```

**Review by thatdocslady — COMMENTED** [review]
Bunch of nit picks (sorry, docs people be docs people ;). Feel free to implement or override as needed.

File: modular-docs-manual/content/topics/creating_procedure_modules.adoc (hunk 1 above)
Comment (emmurphy1) [reply modular-docs-manual/content/topics/creating_procedure_modules.adoc:45]: Suggest changing 'pointing' to 'directing'. I really don't think we should add a Next steps section before the verification. If the 'next steps' are needed to complete the procedure then they should be part of the procedure.

**Comment by mjahoda** [comment]
> Bunch of nit picks (sorry, docs people be docs people ;). Feel free to implement or override as needed.

Thank you, nice improvements. I've just accepted your patch, and I'll just replace "needs to" (because IBM Style discourages its usage similarly as my "should").

File: modular-docs-manual/content/topics/creating_procedure_modules.adoc (hunk 1 above)
Comment (mjahoda) [reply modular-docs-manual/content/topics/creating_procedure_modules.adoc]: > Suggest changing 'pointing' to 'directing'. 

Changed, thank you.

> I really don't think we should add a Next steps section before the verification. If the 'next steps' are needed to complete the procedure then they should be part of the procedure.

However, an example of why we should explicitly allow such order was one of the reasons for filing this issue. Marc Muehlfeld and I believe that in cases such as [1] is this the best order.

[1] https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/creating-and-managing-tls-keys-and-certificates_securing-networks#creating-a-private-ca-using-openssl_creating-and-managing-tls-keys-and-certificates

**Review by ncbaratta — APPROVED** [review]
lgtm

**Pull request merged**